auryadov/02.HTML/birthday.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE HTML>
+<HTML>
+	<HEAD>
+		<META charset="utf-8">
+		<TITLE>Месяц моего дня рождения</TITLE>
+	</HEAD>
+	<BODY>
+		<TABLE align="center" border="3">
+			<CAPTION><I>Месяц моего дня рождения</I></CAPTION>
+			<TR>
+				<TH colspan="6" align="center" valign="bottom">Ноябрь</TH>
+			</TR>
+			<TR>
+				<TH width="40">Пн</TH>
```

Comment:
witch of each columns should be implemented by tag col (see our webinar and presentation)